hack/generate-apiexport-overlays.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Can be executed on registry.redhat.io/openshift4/ose-cli:v4.11
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
+
+if [[ -z ${1} ]]
+then
+  echo "Creates apiexport files in overlays for component"
+  echo "     generate-apiexport-overlays.sh path-to-component-dir"
+  exit 1
+fi
+
+COMPONENT_DIR=$1
+BASE_DIR=$COMPONENT_DIR/base
+
+if [ ! -d $BASE_DIR ]; then
+  echo "component dir must contain 'base' dir"
+fi
+
+TEMP=$(mktemp -d)
+kubectl kustomize $BASE_DIR | csplit -sf $TEMP/resource - /---/ '{*}'
+
+APIEXPORT_FILES=$(grep -lr 'kind: APIExport' $TEMP)
+if [ -n $APIEXPORT_FILES ]; then
+  cat $APIEXPORT_FILES > $COMPONENT_DIR/overlays/dev/apiexport.yaml
+fi
```

Comment:
If the repository contains more than one APIExport, then this could fail. 
```suggestion
mapfile -t APIEXPORT_FILES < <(grep -lr 'kind: APIExport' $TEMP)
if [ ${#APIEXPORT_FILES[@]} != 1 ]; then
  echo "ERROR: the repository contains ${#APIEXPORT_FILES[@]} APIExports. One APIExport per component is supported!";
  exit 1
fi
cat ${APIEXPORT_FILES[0]} > $COMPONENT_DIR/overlays/dev/apiexport.yaml
```